Pyrado/pyrado/algorithms/meta/bayessim.py

```diff
@@ -29,12 +29,14 @@
 from copy import deepcopy
 from typing import Mapping, Optional, Union
 
+import numpy as np
 import torch as to
 from torch.distributions import Distribution
 
 import pyrado
 from pyrado.algorithms.base import Algorithm
 from pyrado.algorithms.inference.snpea import SNPEA
+from pyrado.algorithms.meta.npdr import NPDR
```

Comment:
that is correct, however please do not put in too much work in BayeSim since it will be rewritten

Reply:
Alright, at least pressing Ctrl+Shift+O once wasn't that much effort ;)